src/components/ProductDetail/ProductTabs/InfoTabs.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from "@emotion/styled";
+
+type AnnouncementsItem = {
+  name: string;
+  value: string;
+  displayOrder: number;
+};
+
+type Props = {
+  announcements: AnnouncementsItem[];
+};
+
+export default function InfoTab({ announcements }: Props) {
+  if (!announcements || announcements.length === 0) {
```

Comment:
type에 의하면 announcements가 nullable 하지 않아서 요렇게만 해도 될 것 같네욥!

```suggestion
  if (announcements.length === 0) {
```